src/components/QuickStartPreset.tsx

```diff
@@ -0,0 +1,125 @@
+import { useTranslation } from "react-i18next";
+import React, { useState } from "react";
+interface Params{
+    onClick :(cycles:number)=>void
+}
+const QuickStartPreset = ({onClick}:Params) => {
+      const { t } = useTranslation();
+      const [showCustomOptions, setShowCustomOptions] = useState(false);
+      const [customCycles, setCustomCycles] = useState<number |string>("");
+      const isCustomValid = customCycles !== "" && Number(customCycles) >= 1;
+      {/*const repetitions = Math.floor((minutes * 60) / 19);*/}
+
+
+
+  return (
+    <div className="mt-10 flex flex-col items-center justify-center w-full gap-y-8">
+          <div className="flex flex-wrap justify-center gap-x-4 gap-y-4 md:gap-x-8 items-center">
```

Comment:
Wondering if it the circles should be buttons instead of the divs due to the semantic and accessibility for the page.